<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoad">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWrite">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonContinue">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>550</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
